src/backend/distributed/sql/downgrades/citus--10.1-1--10.0-3.sql

```diff
@@ -20,4 +20,9 @@ DROP FUNCTION pg_catalog.worker_partitioned_relation_total_size(regclass);
 DROP FUNCTION pg_catalog.worker_partitioned_relation_size(regclass);
 DROP FUNCTION pg_catalog.worker_partitioned_table_size(regclass);
 
+#include "../udfs/citus_prepare_pg_upgrade/9.5-1.sql"
 #include "../udfs/citus_finish_pg_upgrade/10.0-1.sql"
+#include "../udfs/get_rebalance_table_shards_plan/9.2-1.sql"
+DROP FUNCTION pg_catalog.citus_add_rebalance_strategy;
+#include "../udfs/citus_add_rebalance_strategy/9.2-1.sql"
```

Comment:
was a bit thrown off by the drop here. As I understand now the `include` below will recreate the function. Guess that version introduced the function? so didn't have a drop of it.

Would it make sense to enforce the creations to have a drop if exists above them, even if it is clearly the first? Although that might be hard with functions that are overloaded.

Maybe adding a comment above the drop, explaining that it will be created by the include is enough. Would also group the drop+include by adding empty lines around them. to signal they belong together.

Reply:
Yes it was the first, one so it didn't have a `DROP` yet. It does do a CREATE OR REPLACE, but since the number of arguments change this doesn't replace existing function. I considered adding `DROP IF EXISTS` to the 9.2-1.sql file, but decided it was better to leave it as is.